### PR TITLE
Client side

### PR DESCRIPTION
**Pull request by Theophileaseh**
Add client-side form validation using javascript

### CODE REVIEW

File: js/validation.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const form = document.getElementById('desktop-form');
+// const firstName = document.getElementById('first-name');
+// const lastName = document.getElementById('last-name');
+// const name = document.getElementById('name');
```
Comment (Deepakdanger) [inline js/validation.js]: - [ ] Let's remove the comment code as it makes the code unclean.

File: js/validation.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  small.innerText = message;
+  formControl.className = 'form-control error';
+  errorMessage.classList.add('smallerror');
+  form.classList.add('formerror');
+}
+
+function checkInputs() {
+  // const firstnameValue = firstName.value.trim();
+  // const lastnameValue = firstName.value.trim();
+  // const nameValue = name.value.trim();
```
Comment (Deepakdanger) [inline js/validation.js]: - [ ] Let's remove the comment code as it makes the code unclean.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Theophileaseh, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] In Readme fle, In `Getting started` section provide the neccesary code to run the projecct locally. For ex:-
```
git clone https://github.com/Theophileaseh/theophileaseh.github.io.git
cd theophileaseh.github.io
```
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
# STATUS: APPROVED :tada:  :confetti_ball: :fireworks: 
Hi @Theophileaseh , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**